frappe/desk/doctype/notification_log/notification_log.py

```diff
@@ -176,7 +176,7 @@ def mark_as_read(docname):
 
 @frappe.whitelist()
 def trigger_indicator_hide():
-	frappe.publish_realtime("indicator_hide", user=frappe.session.user)
+	frappe.publish_realtime("indicator_hide", user=frappe.session.user, after_commit=True)
```

Comment:
```suggestion
	frappe.publish_realtime("indicator_hide", user=frappe.session.user)
```

This might not trigger a commit. Similar problem in other `after_commit=True`